django_redis/client/default.py

```diff
@@ -295,9 +342,7 @@ def delete_pattern(
 
         pattern = self.make_pattern(pattern, version=version, prefix=prefix)
 
-        kwargs = {"match": pattern}
-        if itersize:
-            kwargs["count"] = itersize
+        kwargs = {"match": pattern, "count": itersize}
```

Comment:
These can probably be just inlined if they don't need this conditional checking (They don't on `master`, but not sure if that was the case before)

Reply:
I don't see it anymore